web/templates/job/edit.html.eex

```diff
@@ -0,0 +1,22 @@
+<section class="section">
+  <div class="section-description">
+    <h4>Post a Job</h4>
+    <p>Jobs are free to post and will stay online for 30 days, at which point
+    you'll be sent an email asking if you'd like to extend (for&nbsp;free).</p>
+    <p>Your position should be near Raleigh, NC, and related to UX, UI, or
+    visual design for applications or the&nbsp;web.</p>
+    <%= link(
+      "← Back to Job Board",
```

Comment:
Small nitpick, but maybe `"&laquo; Back to Job Board"` might look better? I'm not sure though you'd have to try it out and see. Whatever you prefer is fine though :)